fs/fs.go

```diff
@@ -1419,7 +1419,7 @@ func NewFs(ctx context.Context, path string) (Fs, error) {
 		extraConfig := overridden.String()
 		//Debugf(nil, "detected overriden config %q", extraConfig)
 		md5sumBinary := md5.Sum([]byte(extraConfig))
-		suffix := base64.RawStdEncoding.EncodeToString(md5sumBinary[:])
+		suffix := strings.Replace(base64.RawStdEncoding.EncodeToString(md5sumBinary[:]), "/", "_", -1)
```

Comment:
I think using `base64.RawURLEncoding` is a better fix here - it uses `-` and `_` as its extra chars so works in filenames.